src/test/java/jenkins/branch/CustomOrganizationFolderDescriptorTest.java

```diff
@@ -76,16 +76,17 @@ public void removeIntegrationTestMocks() throws Exception {
     
     @Test
     public void noNavigatorNoFactoryInstalled() throws Exception {
-        assertEquals(1, ExtensionList.lookup(SCMNavigatorDescriptor.class).size());
-        assertEquals(SingleSCMNavigator.DescriptorImpl.class, ExtensionList.lookup(SCMNavigatorDescriptor.class).get(0).getClass());
-        assertEquals(0, ExtensionList.lookup(MultiBranchProjectFactoryDescriptor.class).size());
+        assertThat(ExtensionList.lookup(SCMNavigatorDescriptor.class),
+                contains(instanceOf(SingleSCMNavigator.DescriptorImpl.class)));
+        assertEquals(0, ExtensionList.lookup(MultiBranchProjectFactoryDescriptor.class).size()); // FIXME fails in PCT
```

Comment:
Currently stuck on these lines (the brittle assertions for the lack of `MultiBranchProjectFactoryDescriptor`). Is there a particular implementation that we could assert is not present?

Reply:
Any of the ones defined as test extensions here—any of the `MockXXX` subtypes.

Reply:
done